tests/test_list_queries.py

```diff
@@ -1,227 +1,220 @@
 import pytest
 import finix
 from finix.models import *
-from finix.configuration import Environment, Configuration
-
-
-@pytest.fixture
-def config():
-    configuration = Configuration(
-        username = 'USsRhsHYZGBPnQw8CByJyEQW',
-        password = '8a14c2f9-d94b-4c72-8f5c-a62908e5b30e',
-        environment = Environment.SANDBOX
-    )
-    return configuration
-
-
-@pytest.fixture
-def config01():
-    configuration = Configuration(
-        username = 'USbkjk46XqUTQHN3i2jaVnc1',
-        password = 'ac915962-2757-49ea-aeee-10960a408b99',
-        environment = Environment.SANDBOX
-    )
-    return configuration
-
-
-@pytest.fixture
-def config02():
-    configuration = Configuration(
-        username = 'USpumes23XhzHwXqiy9bfX2B',
-        password = 'c69d39e3-f9ff-4735-8c3e-abca86441906',
-        environment = Environment.SANDBOX
-    )
-    return configuration
-
-
-@pytest.fixture
-def config03():
-    configuration = Configuration(
-        username = 'USjHFGYvecE4LBitYG8KDE2g',
-        password = 'b698f403-d9b7-4157-82d8-162cea8c8cc3',
-        environment = Environment.SANDBOX
-    )
-    return configuration
-
-
-@pytest.fixture
-def config04():
-    configuration = Configuration(
-        username = 'USimz3zSq5R2PqiEBXY6rSiJ',
-        password = '8bacba32-9550-48ff-b567-fe7648947041',
-        environment = Environment.SANDBOX
-    )
-    return configuration
-
-
-def test_list_transfers(config):
-    client = finix.FinixClient(config)
-    response = client.transfers.list(limit=5)
-    assert isinstance(response, TransfersList)
-    assert response.page['limit'] == 5
-    assert response.embedded['transfers'][0]['id'][:2] == 'TR'
-
-
-def test_list_reversals(config):
-    client = finix.FinixClient(config)
+from finix.model.finix_list import FinixList, PaginationException
+
+
+def test_list_transfers(client00):
```

Comment:
It's hard to parse what exactly is being tested here, breaking the test up and giving descriptive names like `test_list_next_using_offset`, `test_list_next_using_cursor`, `test_list_next_throws_pagination_error` etc would be helpful.

I think it's important to get these tests right and make sure the FinixList object is well tested.

Reply:
make a lot of sense, I will look into this